exercises/practice/scale-generator/.meta/example.rs

```diff
@@ -355,3 +335,23 @@ impl Scale {
         out
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
Test module should appear at the end of a file. There is a pretty fine-grained module structure in this example file. In a real project, this would've been split into several files. But for us, it's most convenient to keep everything in one file.